package.json

```diff
@@ -25,7 +25,7 @@
     "journalism",
     "news"
   ],
-  "author": "Bad Idea Factory <author@biffud.com>",
+  "author": "Bad Idea Factory <biffuddotcom@biffud.com>",
```

Comment:
This email address is a crime, but a delicious crime, like when all that Pappy Van Winkle went missing.

Reply:
I am genuinely assuming this is [a Wishbone reference](https://www.youtube.com/playlist?list=PLDymFoFsHNt14kLH5F3tGy9x5-579qwUq).

Reply:
I just stumbled across this while looking for something else. This is what I get for ignoring all the GitHub notification emails.

No, it was a reference to [this](https://www.huffpost.com/entry/pappy-van-winkle-stolen_n_4114677).